pkgs/hooks/doc/schema/shared/shared_definitions.schema.json

```diff
@@ -261,6 +267,18 @@
       ]
     },
     "LinkOutput": {
+      "type": "object",
+      "properties": {
+        "assets_for_link": {
```

Comment:
This should be moved to the HookOutput not duplicated between LinkOutput and BuildOutput.

Reply:
I feel they are kind of similar, also in the `LinkInput` they should show up as similar/parallel `assetsFromBuid` `assetsFromLink`.